### ISSUE TITLE
Broader flow-bin peer dependency range?

### ISSUE THREAD

**Issue by lydell** [amilajack/eslint-plugin-flowtype-errors#106]
It feels like I always happen to use a non-compatible flow-bin version in my projects, so npm and yarn always warn me about this. Everything still works fine though.

It feels like the interface of the parts of flow-bin that eslint-plugin-flowtype-errors actually use has been stable for a long time, and does not seem to change soon.

Could we change the peer dependency to something like this to avoid unnecessary warnings from npm and yarn?

```
"flow-bin": ">=0.50.0"
```

(Where `0.50.0` is some known compatible version reasonably far back in the past.)

**Comment by amilajack** [comment]
The issue with this is that we test against later flow versions. Cant guarantee that the plugin will work with older versions. I would recommend updating your flow version

**Comment by lydell** [comment]
I understand that you can't guarantee that it will work, but in practice things have never broken for me. Of course I want to use the latest Flow version, but upgrading usually means that new errors are found, so you can't just do it in 5 minutes. And when I _do_ find the time, I probably have upgraded to a newer version than eslint-plugin-flowtype-errors specifies so I get the peer dependency warning anyway...

The peer dependency on ESLint is specified like this:

```
"eslint": ">=4.0.0",
```

Why is that different?
